scripts/servers.py

```diff
@@ -756,7 +757,15 @@ def managed_acceptance_tests_server(
         Exception. The suite_name is not in the list of the acceptance tests
             suite names.
     """
-    if suite_name not in common.ACCEPTANCE_TESTS_SUITE_NAMES:
+    available_suites = {}
+    with open(
+        common.ACCEPTANCE_TEST_CONFIG_FILE_PATH, 'r', encoding='utf-8') as f:
+        filedata = json.load(f)
```

Comment:
We always try to have a change in indentation level in places where we'd normally have an indented block if the lines weren't so long. Here are two ways to do this:

```suggestion
    with open(
        common.ACCEPTANCE_TEST_CONFIG_FILE_PATH, 'r', encoding='utf-8'
    ) as f:
        filedata = json.load(f)
```

or

```suggestion
    with open(
            common.ACCEPTANCE_TEST_CONFIG_FILE_PATH, 'r', encoding='utf-8') as f:
        filedata = json.load(f)
```

Reply:
Done.